usr/share/rear/skel/default/etc/scripts/system-setup.d/55-migrate-network-devices.sh

```diff
@@ -74,7 +98,7 @@ done
 # so that it is shown to the user what MAC address mappings will be done:
 if read_and_strip_file $MAC_MAPPING_FILE ; then
     while read orig_dev orig_mac junk ; do
-        read_and_strip_file $MAC_MAPPING_FILE | grep -q "$orig_mac" && MANUAL_MAC_MAPPING=true
+        read_and_strip_file $MAC_MAPPING_FILE | grep -qw "^$orig_mac" && MANUAL_MAC_MAPPING=true
```

Comment:
I didn't check the details here (i.e. what exactly MAC_MAPPING_FILE might contain)
whether or not the grep with option '-w' will work here in any case as intended, cf.
https://github.com/rear/rear/pull/1897#issuecomment-412024164
and the subsequent comments.

Reply:
MAC_MAPPING_FILE contains this:
~~~
OLD_MACADDR NEW_MACADDR OLD_DEV
~~~

It's undocumented, the script should check for OLD_MACADDR and not just rely on "is the MAC address found" only, hence the `-qw`.

Reply:
@rmetrich 
I guess it is even "documented" to a certain(minimal) extent at
https://github.com/rear/rear/blob/master/doc/mappings/mac.example
;-)